packages/driver/src/cy/snapshots.ts

```diff
@@ -99,15 +101,20 @@ export const create = ($$, state) => {
   }
 
   const getStyles = (snapshot) => {
-    const styleIds = snapshotsMap.get(snapshot)
+    const { ids, styles } = snapshotsMap.get(snapshot) || {}
 
-    if (!styleIds) {
+    if (!ids && !styles) {
```

Comment:
Should this be `||`? It looks like below you are assuming both `ids` and `styles` could be defined.